swagger.yaml

```diff
@@ -49,9 +40,9 @@ securityDefinitions:
 paths:
   /population-types:
     get:
-      tags:
-        - "Public"
-        - "Private"
+      parameters:
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
```

Comment:
Are these params right? I can not see them in code.

Also, do we want to get rid of the public and private tags? If so maybe we should remove them from al endpoints?